runner.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+SCRIPTS_DIR=./sql
+DATABASE_NAME=pesayetu
+
+for file in $SCRIPTS_DIR/*.sql
+    do sudo -u asciicfa psql $DATABASE_NAME -f $file
```

Comment:
Why are we pushing files with _local_ configs?

Reply:
Oh... This code was just something I used to run the scripts locally when we were testing the uppercase files and I was thinking of just keeping the code in the case we need to run all the `SQLs` sometime in the future... @kilemensi